transit_odp/pipelines/pipelines/dataset_etl/transform.py

```diff
@@ -78,6 +93,127 @@ def transform(self, extracted_data: ExtractedData) -> TransformedData:
             service_pattern_stops, service_patterns
         )
 
+        ### logic for flexible stop points transformation
+        if not flexible_stop_points.empty:
+            # 1. extract naptan stoppoints data
+            flexible_stop_points_with_naptan_id = self.sync_flexible_stop_points(
+                flexible_stop_points
+            )
+
+            flexible_stop_points_with_naptan_id = sync_localities_and_adminareas(
```

Comment:
Could we extract the flexible stops past this stage and use the existing sync methods. That is retain the flexible stops in stop_points until method `sync_localities_and_adminareas` and then extract the flexible stop data?

Reply:
done